Pipeline service helm charts [old] (#295)
v3io/helm-charts: stable/pipelines/README.md

> # Kubeflow pipelines operator

Reviewer: ```suggestion
# Kubeflow pipelines service
```

Author: done

Reviewer: You forgot that one

Reviewer: not changed

> Provides installation of the Pipelines job CRDs, RBAC and service, and Mysql as DB

Reviewer: ```suggestion
Provides installation of the Pipelines workflow CRDs, RBAC and services, and Mysql (with fuse backend) as persistency
```

Author: done

## Chart Details

This chart will do one or more of the following:

> * Install Kubeflow Pipelines job CRDs

Reviewer: ```suggestion
* Install Kubeflow Pipelines CRDs
```

Author: done
